apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx

```diff
@@ -123,7 +123,7 @@ function StakeColumn(prop: { stake: bigint; stakePercent: number }) {
                 {prop.stake.toString()}
             </Text>
             <Text variant="captionSmall" color="steel-dark">
-                {prop.stakePercent.toFixed(2)}%
+                SUI
```

Comment:
I don't think this is really true, we need to use `useFormatCoin` to correctly denominate the SUI.

Reply:
Got it

Reply:
is it save to use `0x2::sui::SUI` as the coin type

Reply:
Yeah I think it is